File: qe-selenium-python/Activity19.py
import time
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


with webdriver.Firefox() as driver:

    wait = WebDriverWait(driver, 10)

    driver.get("https://training-support.net/webelements/alerts")

    print("Title of the Page:", driver.title)
    print(driver.title)

    confirmation_button = driver.find_element(By.ID, "confirmation")

    # First alert - accept
    confirmation_button.click()
    wait.until(EC.alert_is_present())

    alert = driver.switch_to.alert
    print("Text in Alert:", alert.text)
    alert.accept()

    time.sleep(2)
    # Second alert - dismiss
    confirmation_button.click()
    wait.until(EC.alert_is_present())

    alert2 = driver.switch_to.alert
    alert2.dismiss()

File: qe-selenium-java/Activity19.java
package demos;

import java.time.Duration;
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity19 {
public static void main(String[] args) {
    WebDriver driver = new FirefoxDriver();
    WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
    driver.get("https://training-support.net/webelements/alerts");
    System.out.println("Title of the Page: "+driver.getTitle());

    System.out.println(driver.getTitle());

    WebElement confirmationButton = driver.findElement(By.id("confirmation"));
    confirmationButton.click();

    wait.until(ExpectedConditions.alertIsPresent());

    Alert alert = driver.switchTo().alert();

    System.out.println("Text in Alert: "+alert.getText());

    alert.accept();

    
    confirmationButton.click();
    wait.until(ExpectedConditions.alertIsPresent());
    Alert alert2 = driver.switchTo().alert();

    alert2.dismiss();
    // Close browser
    driver.quit();
    }
}
    


